lib/cart.rb

```diff
@@ -0,0 +1,59 @@
+require_relative "menu_item"
+require_relative "menu"
+require_relative "order_item"
+require_relative "customer"
```

Comment:
As I understand it, 'require_relative' is to load files not in Ruby's default load path.  I used it because it was used in the example I was following, but I should have tested if I had to do it that way.  Thank you for calling that to my attention.  I did, just now, research it on Stachoverflow: 

"require_relative complements the builtin method require by allowing you to load a file that is relative to the file containing the require_relative statement.

For example, if you have unit test classes in the "test" directory, and data for them under the test "test/data" directory, then you might use a line like this in a test case:
`require_relative "data/customer_data_1"`